packages/RecentChangesAPI/src/custom/modules/Users/clients/base/api/CustomRecentChangesApi.php

```diff
@@ -151,10 +153,27 @@ protected function validateParam($paramName, $paramValue, $constraint){
      * @return mixed The validated param
      */
     protected function validateSinceParam($sinceParam){
-        $dateConstraint = new Assert\DateTime(array(
-            'format' => $this->dateFormat,
-            'message' => "Param 'since' must be of the format '$this->dateFormat'"));
-        return $this->validateParam("since", $sinceParam, $dateConstraint);
+        try{
+            # The DateTime constraint with a format option is available beginning with Symfony 3.1
+            # If possible, we want to use the DateTime constraint
+            $dateConstraint = new Assert\DateTime(array(
+                'format' => $this->dateFormat,
+                'message' => "Param 'since' must be of the format '$this->dateFormat'"));
+            return $this->validateParam("since", $sinceParam, $dateConstraint);
+        }
+            # If the DateTime constraint is not available, we will validate in a different way
```

Comment:
In reality, we need this catch because `format` is not a supported parameter in older versions of Symfony DateTime constraint.